shared/test/police_box.rb

```diff
@@ -0,0 +1,50 @@
+require 'timecop'
+
+#
+# PoliceBox - Because sometimes tests are timey-wimey.
+#
+# This is a utility for tests that should use fake time using Timecop in some
+# circumstances (like when running in true isolation) and real time in other
+# circumstances (like when running against a real database).
+#
+class PoliceBox
+  def initialize(in_real_time)
+    @time_circuits_on = !in_real_time
```

Comment:
Can this (`@time_circuits_on`) be better named? Without looking, I have no idea whether this indicates we are using real time or fake time. Also, it smells that you are changing the name from `in_real_time` (negated) to `@time_circuits_on`.

Maybe just `use_fake_time`?

Aside, I'm not a fan of `PoliceBox`.  The person next to you probably has no idea what this class does. Similarly for `PoliceBox#in_box`. Granted, I don't have better, so no change required here.